tests/cases/test_parser.py

```diff
@@ -77,6 +77,50 @@ def test_process_autocorrelation(self):
             if process:
                 process.purge()
 
+    def test_process_gamma(self):
```

Comment:
@chuehlien @mbertrand i am wondering why we are adding these tests to test_parser.py file. Shouldn't we add them to the test_processes.py?